miniwallet/config.ts

```diff
@@ -4,10 +4,26 @@ import 'dotenv/config'
 export default {
   mainnet: {
     miniWallet: {
-      initialOperatorThreshold: process.env.MINIWALLET_INITIAL_OPERATOR_THRESHOLD,
+      initialOperatorThreshold: process.env.MINIWALLET_INITIAL_OPERATOR_THRESHOLD || '100',
       initialOperators: JSON.parse(process.env.MINIWALLET_INITIAL_OPERATORS || '[]'),
       initialUserLimit: ethers.utils.parseEther(process.env.MINIWALLET_INIITIAL_USER_LIMIT || '1000000'),
       initialAuthLimit: ethers.utils.parseEther(process.env.MINIWALLET_INIITIAL_AUTH_LIMIT || '100000')
     }
+  },
+  ethlocal: {
+    miniWallet: {
+      initialOperatorThreshold: process.env.ETH_LOCAL_MINIWALLET_INITIAL_OPERATOR_THRESHOLD || '10',
+      initialOperators: JSON.parse(process.env.ETH_LOCAL_MINIWALLET_INITIAL_OPERATORS || '[]'),
+      initialUserLimit: ethers.utils.parseEther(process.env.ETH_LOCAL_MINIWALLET_INIITIAL_USER_LIMIT || '1000'),
+      initialAuthLimit: ethers.utils.parseEther(process.env.ETH_LOCAL_MINIWALLET_INIITIAL_AUTH_LIMIT || '100')
+    }
+  },
+  hardhat: {
```

Comment:
what's the difference between ethLocal and hardhat config?

Reply:
ethLocal is used for end to end testing. It is part or the local testing environment documented [here](https://github.com/polymorpher/sms-wallet/blob/main/miniwallet/devlog/FEATURE_ROLLOUT.md#launching-the-infrastructure) specifically using ganache as documented [here](https://github.com/polymorpher/sms-wallet/blob/main/miniwallet/devlog/FEATURE_ROLLOUT.md#gancache-port8545)

hardhat config is used by the following commands
* [yarn test](https://github.com/polymorpher/sms-wallet/blob/main/miniwallet/package.json#L24) or `npx hardhat test` 
* [yarn deploy](https://github.com/polymorpher/sms-wallet/blob/main/miniwallet/package.json#L19) or `px hardhat deploy --tags deploy`

Note the tags are populated in each of the test scripts
e.g. in [001_deploy_miniWallet.ts](https://github.com/polymorpher/sms-wallet/blob/main/miniwallet/deploy/001_deploy_miniWallet.ts#L67) we use the following tags `deployFunction.tags = ['MiniWallet', 'deploy', 'MiniWalletHardhatDeploy']`
There are additional control mechanisms which can be leveraged for sequencing if needed.